src/features/vars-schema/env-vars.ts

```diff
@@ -267,6 +267,12 @@ export const DEVICE_TYPE_SPECIFIC_CONFIG_VAR_PROPERTIES: Array<{
 					'Define the file name of the DTB to be used. Only supported by supervisor versions >= v11.14.2.',
 				will_reboot: true,
 			},
+			BALENA_HOST_EXTLINUX_isolcpus: {
```

Comment:
Atm we have about 3.5k records using the `RESIN_HOST_EXTLINUX_isolcpus` and none using the `BALENA_` one.
As a result using `BALENA_HOST_EXTLINUX_isolcpus` would still make their config var show as a custom one and will not show up in the schema generated UI form.
Moreover, since we state that this was added in `v7.10.0` and we only started supporting `BALENA_` in v7.25.0, this could potentially cause confusion if anyone tries to use it in v7.10.0.
See: https://github.com/balena-os/balena-supervisor/blob/master/CHANGELOG.md#v7250---2018-10-18

Maybe we should use `RESIN_HOST_EXTLINUX_isolcpus` for now and wait for the RESIN_ & BALENA_ unification to take care of all prefixes in one go.

Reply:
Right on, will change the supported by field